### PR TITLE
Updated to proper URL that works for IPTorrents

### PR DESCRIPTION
**Pull request by 97WaterPolo**

### CODE REVIEW

File: lib/providers/iptorrents.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "IpTorrents",
-    "baseUrl": "https://iptorrents.eu",
+    "baseUrl": "https://iptorrents.com",
```
Comment (zakstack) [inline lib/providers/iptorrents.json]: Did this actually manage to get this working for you? I haven't been able to get any search results regardless of if it's er or com

Comment (97WaterPolo) [reply lib/providers/iptorrents.json]: Yes I did. Here is a snip of the code for my IPTorrents, and I got it to work. This is my iptorrents.json file located at {PROJECT} > node_modules > torrent-search-api > lib > providers. https://gist.github.com/97WaterPolo/3bcbb9fb61b50746af11604676924be3

To run it, I just used. 
const TorrentSearchApi = require('torrent-search-api'); //TO search for torrents
TorrentSearchApi.enableProvider('IpTorrents', 'USERNAME', 'PASSWORD');
async function startTorrentQuery(searchString, category, amount){
    return await TorrentSearchApi.search(searchString, category, amount);
}

Comment (zakstack) [reply lib/providers/iptorrents.json]: Odd. I'm not able to get it to search even with this code. No errors either just blank arrays being returned.

Would you be able to test it and see if it still works for you?

Comment (97WaterPolo) [reply lib/providers/iptorrents.json]: Sure send the GIST and I'll test it

Comment (JimmyLaurent) [reply lib/providers/iptorrents.json]: Hi guys,

IpTorrents url may be blocked by your isp, you can override it

```js
torrentSearch.overrideConfig("Iptorrents", {
  baseUrl: "IPTORRENTS_PROXY_URL"
});
```

**Pull request closed without merging**